app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TasksController < ApplicationController
+
+  # name : string
+  # description : string
+  # completion_date : string?
+  # done : boolean
+
+  def index
+    @tasks = Task.order(:done)
+  end
+
+  def show
+    @task = Task.find(params[:id].to_i)
```

Comment:
I would suggest anytime you try to find something from a database you have code to handle when that item isn't there.  So some sort of if statement.